frontend/src/pages/archive/crawl-detail.ts

```diff
@@ -318,23 +323,19 @@ export class CrawlDetail extends LiteElement {
   private renderFiles() {
     return html`
       <ul class="border rounded text-sm">
-        ${this.crawl?.files?.map(
+        ${this.crawl?.resources?.map(
           (file) => html`
             <li class="flex justify-between p-3 border-t first:border-t-0">
               <div>
                 <a
                   class="text-primary hover:underline"
-                  href=${file.filename}
+                  href=${file.path}
                   download
-                  title=${file.filename.slice(
-                    file.filename.lastIndexOf("/") + 1
-                  )}
+                  title=${file.name}
                   >${msg(
-                    str`Download ${file.filename.slice(
-                      file.filename.lastIndexOf(".")
-                    )}`
-                  )}</a
-                >
+                    str`Download ${file.name}`
```

Comment:
Thoughts on showing just the file name and not the whole path?
```suggestion
                    str`Download ${file.name.slice(
                      file.name.lastIndexOf("/") + 1
                    )}`
```

Looks currently like this:
<img width="1040" alt="Screen Shot 2022-01-31 at 12 09 42 PM" src="https://user-images.githubusercontent.com/4672952/151865362-7e552f5b-4f3e-4f3c-9f53-6c0a64c3f77b.png">

Trimmed down:
<img width="1034" alt="Screen Shot 2022-01-31 at 12 10 38 PM" src="https://user-images.githubusercontent.com/4672952/151865474-6020ccda-7f83-4fc2-96d6-653649ff8372.png">

Reply:
Ah sure, yeah, that makes sense! And maybe should also move download into the title